README.md

```diff
@@ -160,6 +160,8 @@ RailsApiAuth.tap do |raa|
   raa.google_client_secret = '<your Google client secret>'
   raa.google_redirect_uri = '<your app redirect uri>'
 
+  # Force SSL for Oauth2Controller; defaults to `true`
+  raa.force_ssl = false if Rails.env.development?
```

Comment:
My way is more expressive and easier on the brain :) ; your way is shorter. I am OK with either way - you tell me which one you want me to do.